### DISCUSSION TITLE
the coins did not arrive to the wallet

### DISCUSSION THREAD

**Discussion by m-artur** [Chia-Network/chia-blockchain#18512] [Support]
https://xchscan.com/address/xch1pmq5cv9l8xrgsu2wd6zfwca045elwkv9up69nsjz97rpy5aj73pqsyy853
the coins did not arrive to the wallet: xch1pmq5cv9l8xrgsu2wd6zfwca045elwkv9up69nsjz97rpy5aj73pqsyy853
what to do? When will the coins arrive?

**Comment by BrandtH22** [comment]
Hey @m-artur  , as a heads up we can generally provide more timely, thorough support in our discord server (https://discord.gg/chia).

What I recommend trying is a resync of the wallet:

- If using the gui go to settings -> advanced -> resync . This will close the client and upon reopening it will resync.
- If using the cli then I would stop chia, rename the wallet db folder in ~.chia\mainnet\wallet\db then restart chia.

If the funds are still not visible can you let us know:

- if the wallet icon in the top right is orange or green?
- if orange can you click on the wallet icon and let us know if you see any connected peers?
- can you provide your log file from ~.chia\mainnet\log\debug.log?

**Comment by m-artur** [comment]
@BrandtH22   I did everything, it didn't help. The wallet is Orange, but there are no connected users

**Reply by BrandtH22** [reply]
@m-artur what version of chia are you running?

**Reply by m-artur** [reply]
@BrandtH22  Chia: 2.4.3

**Comment by m-artur** [comment]
debug.log   where can I drop it? it doesn't work here in the comment

**Reply by BrandtH22** [reply]
You should be able to drop the entire file here or copy its contents and paste it here :+1:

**Reply by m-artur** [reply]
https://disk.yandex.ru/d/1BXqIBY5Eh-YCQ

**Comment by BrandtH22** [comment]
Hey @m-artur , thank you for the logs. As a reminder we can provide more thorough and timely support in our discord server (https://discord.gg/chia). 

The logs indicate a DNS issue (networking issue). To help identify the cause of the issue can you:
- run the command `nslookup dns-introducer.chia.net` and provide a screenshot of the output?
- also run the command `nslookup dns-introducer.chia.net 8.8.8.8` and provide a screenshot of the output?

**Reply by m-artur** [reply]
![image](https://github.com/user-attachments/assets/addf3a65-c71a-4e21-81a3-d2ffcbd32afa)

**Reply by m-artur** [reply]
> dns-introducer.chia.net 8.8.8.8
╤хЁтхЁ:  [8.8.8.8]
Address:  8.8.8.8

Не заслуживающий доверия ответ:
╚ь :     dns-introducer.chia.net
Addresses:  2603:3023:69b:e100:694a:7fc7:ce13:d96
          2403:6200:8851:21b4:f07d:73c5:50a5:3909
          2001:b011:70a0:1115:8c0e:d70b:72c2:6495
          2001:871:265:5275:d1b8:7b00:df9:e9a
          2a01:e0a:3d2:bf00:d4f4:9156:8ab9:a778
          2403:6200:8837:527:1818:73cb:3760:6955
          2a01:e11:5401:bab0:88bb:f8f5:c6b5:83c1
          2a01:e0a:369:6760:892a:d408:67ad:a828
          2a13:8a02:462f:5500:f894:2a3:5c60:7edf
          240e:3a1:1661:6d70:e983:9b6b:a43a:c141
          2603:3015:10d9:6000:9c48:1f56:8f1b:c423
          2a02:a58:8340:6a00:90e4:3dc4:abfa:2d63
          2a00:1370:81a4:ffd:e836:2d34:4f28:bb7a
          2806:104e:1f:1626:789a:8885:6407:a0d3
          2806:269:483:440:eb96:f5a5:5cb5:d49c
          2001:f40:960:f4e:5083:7a26:4718:1ec0
          2a01:e0a:360:7910:f1ad:901c:fd3c:1f71
          2a02:121e:7290:0:c1e4:73b8:fcc2:f5c0
          2806:2f0:9200:1c1c:c48d:9640:51be:da3d
          2001:da8:ff:212::ff:57
          2001:fb1:76:e915:a459:5ca8:b40:5cd2
          2800:bf0:2c04:1078:b1ef:dc30:3078:d3a2
          2a01:e0a:d0d:2100:c3c:b94a:2109:b286
          240e:39a:ed3:43f0:c9f9:bb3c:2a73:1aae
          2406:3003:2003:2488:c506:9404:8ea1:ee1
          2806:104e:1f:1626:b0b7:3e0f:cf64:e1a3
          2a01:e11:7004:8100:e46d:6d29:2a5:8434
          2603:3001:1914:b100:660d:f1a:9169:91fe
          2800:40:39:a5:5456:6cab:9346:5927
          2a00:1858:104d:90ef:8d7a:dd11:d61b:a058
          2403:6200:8881:de43:21de:3353:6262:ff76
          240e:3af:21f:ad1:35e5:2c10:cc7d:8149
          114.230.221.11
          70.143.114.147
          212.55.84.66
          85.207.37.6
          89.76.52.130
          50.47.131.208
          77.51.217.59
          201.21.186.26
          92.206.222.234
          203.164.244.230
          86.125.246.178
          93.94.243.190
          46.42.11.69
          67.5.24.215
          98.61.204.92
          193.237.139.12
          122.137.135.204
          125.46.171.29
          92.233.118.170
          114.230.221.4
          83.58.219.202
          213.200.46.49
          85.194.204.20
          96.27.59.12
          86.5.203.68
          188.239.99.134
          42.84.34.110
          94.45.205.117
          94.102.126.26
          5.55.102.59
          76.206.62.137
          72.131.113.82

**Reply by m-artur** [reply]
![image](https://github.com/user-attachments/assets/60648d8c-4b60-49ec-b3a4-19605027b06e)

**Reply by m-artur** [reply]
![image](https://github.com/user-attachments/assets/6dd3db03-fe0a-4574-91eb-c6c37ea70f48)

**Reply by BrandtH22** [reply]
@m-artur can you try updating your dns server?

### Windows 10
- Right-click the Windows icon in your toolbar and select Settings.  
- Then select Network & internet.  
- Assuming you are using an Ethernet connection, click the Ethernet option. For Windows 10 you will click the "Change adapter options" selection and then right-click your Ethernet connection on the resulting page.  If you are using a wireless connection right click on the wireless connection instead.
- From the drop-down, select Properties.  
- In the Ethernet Properties window, highlight "Internet Protocol Version 4" and then click the Properties button.  
- Select the "Use the following DNS server addresses" option
- enter 8.8.8.8 for the Preferred DNS and for the Alternate DNS enter 1.1.1.1 and then click Save.  

This will manually set you up to use Google (8.8.8.8) and Cloudflare (1.1.1.1) as your DNS providers.

(Note: if you have "Internet Protocol Version 6 (TCP/IPv6)" enabled in your network properties you will also want to adjust the Preferred/Alternate DNS server settings for that protocol as well.  In the Ethernet Properties windows, highlight "Internet Protocol Version 6" and then click the Properties button.  Select the "Use the following DNS server addresses" option and then enter 2001:4860:4860::8888 for the Preferred DNS and for the Alternate DNS enter 2606:4700:4700::1111 and then click Save)

The end result should look like this:

![image](https://github.com/user-attachments/assets/634e9067-2c09-47cb-aa94-628b8bd85be4)

Once you have this configured, please reboot your PC to ensure the new settings take effect.  Once the reboot is complete, please start up your Chia wallet and allow it a few moments to sync.  You can keep an eye on your Wallet icon and click on it to see if you have any active connections.  If after 10-15 minutes you still don't have any wallet connections and your icon is still orange, please copy/paste your debug.log here for another review

### Windows 11

To adjust your DNS server settings on Windows 11:
1) Right-click the Windows icon in your taskbar and select the Settings option
2) Your network connection(s) will be listed at the top of the next screen (example below). Click the Properties option next to your active connection.
![image](https://github.com/user-attachments/assets/ea38423a-e18f-450f-b08e-767356c053e0)
3) Click the Edit button next to the "DNS server assignment" option.
![image](https://github.com/user-attachments/assets/16383d97-6752-40ed-a212-2d3b93c397ff)
4) Toggle the dropdown from Automatic to Manual and then toggle the "IPv4" button to ON.  Set the Preferred DNS to 8.8.8.8 and the Alternate DNS to 1.1.1.1. 
![image](https://github.com/user-attachments/assets/977c5e7b-92d9-445a-8174-c776358d4f1d)

This will set your preferred DNS server to Google (8.8.8.8) and your alternate to Cloudflare (1.1.1.1). 
5) Do the same for the "IPv6" toggle option, except the addresses will be different.  Set the Preferred DNS to 2001:4860:4860::8888 and the Alternate DNS to 2606:4700:4700::1111 and then click Save.

Once you have this configured, please reboot your PC to ensure the new settings take effect.  Once the reboot is complete, please start up your Chia wallet and allow it a few moments to sync.  You can keep an eye on your Wallet icon and click on it to see if you have any active connections.  If after 10-15 minutes you still don't have any wallet connections and your icon is still orange, please copy/paste your debug.log here for another review

**Reply by m-artur** [reply]
Thanks, it's fine.

**Reply by BrandtH22** [reply]
Thank you for confirming that these steps resolved the issue. 
Let us know if any other questions arise and make it an excellent day,
Brandt H
